src/components/QuranReader/index.tsx

```diff
@@ -38,18 +40,19 @@ const verseFetcher = async (input: RequestInfo, init?: RequestInit) => {
 
 const QuranReader = ({ initialData, chapter }: QuranReaderProps) => {
   const quranReaderStyles = useSelector(selectQuranReaderStyles);
+  const currentTranslations = useSelector(selectCurrentTranslations) as string[];
```

Comment:
why do you need the `as string[]` part? It should be done automatically by typescript, no?

Reply:
No it doesn't resolve it automatically, it shows `any` as the type.